nuxt/index.js

```diff
@@ -52,21 +52,39 @@ module.exports = function nuxtBootstrapVue(moduleOptions = {}) {
       this.options.css.unshift('bootstrap/dist/css/bootstrap.css')
     }
 
-    // Transpile src/
+    // Component src prop resolving
+    this.options.build.loaders.vue.transformAssetUrls = {
+      // Ensure defaults are not lost
+      ...this.options.build.loaders.vue.transformAssetUrls,
+      // Nuxt default is missing `poster` for video
+      video: ['src', 'poster'],
```

Comment:
Can be `(this.options.build.loaders.vue.transformAssetUrls || []).concat('src', 'poster')`

Reply:
And for simplification extracting `transformAssetUrls` before L55:

```js
const { transformAssetUrls } =  this.options.build.loaders.vue
```